Tests/FluentAssertions.Specs/Equivalency/CollectionEquivalencySpecs.cs

```diff
@@ -1489,7 +1489,7 @@ public void When_nested_objects_are_excluded_from_collections_it_should_use_simp
             IList<MyObject> expectationList = new List<MyObject> { expectation };
 
             // Act
-            Action act = () => actualList.Should().BeEquivalentTo(expectationList, opt => opt.ExcludingNestedObjects());
+            Action act = () => actualList.Should().BeEquivalentTo(expectationList, opt => opt.ComparingByMembers<MyChildObject>().ExcludingNestedObjects());
```

Comment:
Has `BeEquivalentTo` changed behavior in detecting value semantics?

Reply:
That test wasn't actually testing what it was supposed to do ;-)